lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -86,7 +91,8 @@ const useFetchGrade = student => {
  */
 export default function SubmitGradeForm({ student }) {
   // State for loading the grade
-  const { grade, gradeLoading } = useFetchGrade(student);
+  const [fetchGradeError, setFetchGradeError] = useState('');
```

Comment:
To be consistent with the rest of the previous code this line should be inside the `useFetchGrade` and the `fetchGradeError` should be returned from that function.

Reply:
Thats was the main issue I had in the first place. I'm not convinced encapsulating useFetchGrade is working for us anymore because I need to call setFetchGradeError when the cancel button is pressed in order to close out the dialog. But I also need to call it inside of useFetchGrade when an error is first detected. I supposed the right thing to do is to get rid of the custom effect unless there is a simpler solution I'm not seeing?